lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1721,11 +1741,15 @@ private Response makeRequest(
             final String urlPath,
             final Map<String, String> attributes,
             final HttpEntity body)
-            throws IOException {
+            throws IOException, IndexingConflictException {
         try {
             return getRestClient().performRequest(reqType, urlPath, attributes, body);
-        } catch (ResponseException ex) {
-            throw new IOException(ex.getMessage(), ex);
+        } catch (ResponseException re) {
+            if (re.getResponse().getStatusLine().getStatusCode() == 409) {
+                // this is really difficult to test, and so is not
```

Comment:
comment looks incomplete? and so is not...

Reply:
fixed